docs/security-considerations.rst

```diff
@@ -0,0 +1,208 @@
+#######################
+Security Considerations
+#######################
+
+While it is usually quite easy to build software that works as expected,
+it is much harder to check that nobody can use it in a way that was **not** anticipated.
+
+In Solidity, this is even more important because you can use smart contracts
```

Comment:
Break this into multiple sentences? It feels like it's packing too many ideas into one sentence.